selenium/integration/admin.spec.ts

```diff
@@ -47,6 +47,7 @@ describe('Setting and saving preferences', () => {
     await driver.get(CONFIGURATION_URL);
     await driver.sleep(1000);
     await driver.findElement(By.css(dataTestId('system-prefs-accordion'))).click();
+    await driver.sleep(200);
```

Comment:
I don't think selenium likes consecutive clicks, especially if it triggers an animation